package.nls.tr.json

```diff
@@ -0,0 +1,90 @@
+{
+    "bookmarks.activitybar.title": "Yer İşaretleri",
+    "bookmarks.views.Explorer.name": "Tarayıcı",
+    "bookmarks.views.HelpAndFeedback.name": "Yardım ve Geri Bildirim",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenEmpty": "Yer İşaretlerini kullanmak için önce bir klasör veya çalışma alanı açmanız gerekir.\n[Klasör Aç](command:_bookmarks.openFolderWelcome)\n[Çalışma Alanı Aç](command:workbench.action.openWorkspace)\nDaha fazla bilgi edinmek için VS Code'da Yer İşaretlerinin nasıl kullanılacağı hakkında [belgeleri okuyun](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenNoFileOpen": "Henüz yer işareti yok.\nYer İşaretlerini kullanmak için düzenleyicide bir dosya açmanız gerekir.\n[Dosya Aç](command:workbench.action.quickOpen)\nYer İşaretlerini VS Code'da nasıl kullanacağınız hakkında daha fazla bilgi edinmek için [dokümanları okuyun](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenHasFileOpen": "Henüz yer işareti yok.\nYer İşaretlerini kullanmak için imleci dosyada herhangi bir konuma getirin ve şu komutu çalıştırın:\n[Yer İşaretleri: Geçiş](komut:bookmarks.toggle)\nVS Code'da Yer İşaretlerinin nasıl kullanılacağı hakkında daha fazla bilgi edinmek için [belgeleri okuyun ](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
```

Comment:
Shouldn’t be `command:bookmarks.toogle` instead of `komut:bookmarks.toggle`? I think the `command` keyword should be translated here, as in the other commands